### ISSUE TITLE
Ability to quickly navigate from the path at the top of the panel under the tab

### ISSUE THREAD

**Issue by nitrostudios** [mikekazakov/nimble-commander#110]
Would it be possible to have the ability to click on the name in the path to quickly change the directory to that location, please?

E.g.: /Library/Application Support/Apple/Automator/Workflows/

Clicking on Application Support will change to that location.

Thank you for consideration.

![Screen Shot 2024-01-27 at 15 18 26](https://github.com/mikekazakov/nimble-commander/assets/157495999/424e6134-af5c-441f-b07f-057c884a7ce4)

**Comment by vladstudio** [comment]
Hey,

I think you can already do that using Go / Quick lists / Parent folders (you can set a hotkey of your choice in Settings).

**Comment by mikekazakov** [comment]
Absolutely, I'd love to see that feature in NC too. Just gotta figure out when I can squeeze in some time and muster the energy to get it done! 😉

**Comment by nitrostudios** [comment]
That's great, Mike, thank you! Fingers crossed and looking forward:)
